images/dockerregistry/config.yml

```diff
@@ -35,3 +35,8 @@ middleware:
         blobrepositorycachettl: 10m
   storage:
     - name: openshift
+openshift:
+  version: 1.0
+  metrics:
+    enabled: false
+    secret: TopSecretToken
```

Comment:
same problem :-) how about not having it enabled by default and update `oc registry` to allow to set this up?